### PR TITLE
Expose getpreviousaddresses/undo twofactor reset

### PR DESCRIPTION
**Pull request by jgriffiths**
Needs gdk to be updated before merging

### CODE REVIEW

File: green_cli/green.py [hunk 1]
```diff
@@ -495,6 +495,15 @@ def getreceiveaddress(session, details):
...
     return gdk.get_receive_address(session.session_obj, json.dumps(details))
 
+@green.command()
+@click.argument('subaccount', type=int)
+@click.argument('last_pointer', type=int)
+@with_login
+@print_result
+def getpreviousaddresses(session, subaccount, last_pointer):
+    """Get previously generated addresses"""
+    return json.loads(gdk.get_previous_addresses(session.session_obj, subaccount, last_pointer))
```
Comment (jkauffman1) [inline green_cli/green.py:505]: Can be written:

`return session.get_previous_addresses(subaccount, last_pointer)`

File: green_cli/green.py [hunk 2]
```diff
@@ -495,6 +495,15 @@ def getreceiveaddress(session, details):
     """Get a new receive address"""
     return gdk.get_receive_address(session.session_obj, json.dumps(details))
 
+@green.command()
+@click.argument('subaccount', type=int)
```
Comment (jkauffman1) [inline green_cli/green.py:499]: In other calls subaccount defaults to 0

File: green_cli/green.py (hunk 1 above)
Comment (jgriffiths) [reply green_cli/green.py:505]: Changed to a Call interface for liquid now.

**Comment by jkauffman1** [comment]
utack fcd1a52eb66d42dc8a3b11238304899edbd45e3f

**Pull request merged**